examples/address_generator/src/main.rs

```diff
@@ -0,0 +1,134 @@
+//
+// Copyright 2019 Cargill Incorporated
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use clap::{App, Arg};
+
+use transact::contract::address::{
+    double_key_hash::DoubleKeyHashAddresser, key_hash::KeyHashAddresser,
+    triple_key_hash::TripleKeyHashAddresser, Addresser,
+};
+
+fn main() -> Result<(), String> {
+    let matches = App::new(clap::crate_name!())
+        .version(clap::crate_version!())
+        .author(clap::crate_authors!())
+        .about(clap::crate_description!())
+        .arg(
+            Arg::with_name("prefix")
+                .required(true)
+                .takes_value(true)
+                .help("Prefix of the radix address.")
+                .index(1),
+        )
+        .arg(
+            Arg::with_name("key")
+                .required(true)
+                .multiple(true)
+                .takes_value(true)
+                .max_values(3)
+                .index(2)
+                .help(
+                    "Natural key used to compute the radix address.\nMust supply at least 1 key.",
+                ),
+        )
+        .arg(
+            Arg::with_name("length")
+                .required(false)
+                .max_values(2)
+                .last(true)
+                .index(3)
+                .help(
+                    "Optional length of hashes used to construct the radix address.\nMust be u32.",
```

Comment:
Remove newline and 2nd sentence; u32 holds no meaning to sysadmins, and it is already obvious it will be a number. Remove "Optional" from the description, this will be clear when it is a non-positional flag.